config.py

```diff
@@ -9,7 +9,7 @@
 DEBUG = False
 
 SERVICE_ROOT = 'https://waterservices.usgs.gov'
-LEGACY_ENDPOINTS = {
+CGI_ENDPOINTS = {
```

Comment:
I don't like calling these CGI_ENDPOINTS. Maybe we could say NWIS_ENDPOINTS?

Reply:
Sure